scale-tests/sdk.dashboard.py

```diff
@@ -0,0 +1,266 @@
+"""
+Generate a generic SDK dashboard.
+
+```bash
+pip install grafanalib
+
+generate-dashboard -o sdk.json sdk.dashboard.py
+```
+
+This creates an `sdk.json` file that contains the Grafana dashboard definition for monitoring SDK-based frameworks
+installed on a Cluster.
+"""
+
+from grafanalib.core import *
```

Comment:
Please enumerate the pieces you need. `*` is considered bad style because it's a maintenance headache.